### ISSUE TITLE
typo in OPL-update

### ISSUE THREAD

**Issue by pstaabp** [openwebwork/webwork2#958]
I'm working on something related to OPL update and my script was getting the error:
```
Global symbol "$ENABLE_UTF8MB" requires explicit package name (did you forget to declare "my $ENABLE_UTF8MB"?) at updateOPLextras line 50.
Execution of updateOPLextras aborted due to compilation errors.
```
 (line number is my scripts line number). 

but I think we should be getting the same error on OPL-update line 102.   I think it is a typo and should be $ENABLE_UTF8MB4. 

Perhaps this is because we're not using `use warnings` in OPL-update -- I am in my script.  Also, my OPL-update runs fine, but perhaps this is because I haven't tested any of the UTF8 in the database.

**Comment by taniwallach** [comment]
It is certainly a typo, and the only real effect of the type is that even when the course environment was set up to allow the full UTF-8 (what mySql calls `utf8mb4`), that would not get enabled in OPL-update. The difference is mainly in special characters like emoji, as just about any reasonable character (including almost all foreign language characters) apparently fit into 3 UTF-8 bytes. 

I just made https://github.com/openwebwork/webwork2/pull/960 with a fix for the WeBWorK-2.15 branch and before I could finish this post @mgage has merged it in to that branch.

The need for the control switch is to support older version of mySQL and Perl which do not have the newer capabilities.

I'm not totally sure how the "weaker" Perl DBD setting `mysql_enable_utf8` will really handle 4-byte UTF-8 characters... They may actually pass-through and just hit problems if the backend database is using the 3-byte `utf8` character set in the database itself.

**Comment by mgage** [comment]
I think you may have fixed this already.  I don't see a reference to $ENABLE_UTF8MB4 near line 50
of updateOPLextras.  I do see a reference to $ce->{ENABLE_UTF8MB4} which should be defined
in the course environment (in site.conf if I remember correctly or defaults.config)

I'll fix the typo on line 102 of develop. Tani has already fixed it for WeBWorK-2.15.
